frontend-student/src/components/allcourses/Courses.jsx

```
@@ -28,50 +28,57 @@ export default function Courses() {
+              <img
+                src={getThumbnail(course.thumbnail)}
+                alt={course.title}
+                className="w-full h-full object-cover"
+              />
+            </div>
+            <div className="p-4">
+              <h2 className="text-xl font-semibold">{course.title}</h2>
+            </div>
+            <div className="p-4 flex justify-between items-center border-t">
+              <Link to={`/courses/${course.id}`}>
+                <button className="px-4 bg-blue-500 text-white py-3 rounded-lg hover:bg-blue-600 transition">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)